site/content/contribute/getting-started/core-committers.md

```diff
@@ -56,6 +56,9 @@ Below is the list of core committers working on Mattermost:
 - **Hanzei**
     - @hanzei on [community.mattermost.com](https://community.mattermost.com/core/messages/@hanzei) and [@hanzei](https://github.com/hanzei) on GitHub
     - Dev areas: Plugins, Integrations, Security
+- **Lev Brouk**
+    - @lev.brouk on [community.mattermost.com](https://community.mattermost.com/core/messages/@lev.brouk) and [@levb](https://github.com/levb) on GitHub
+    - Dev areas: Developer toolkit, performance
```

Comment:
I'm know I'm a little to late, but it might be worth to keep this consistent and use `Plugins, Integrations` instead of `Developer toolkit`. 2/5